### ISSUE TITLE
"No such driver version" error on Windows 10, Chrome 115.x

### ISSUE THREAD

**Issue by nolasaint** [chilli-axe/mpc-autofill#157]
## System Info
Release: `v4.3.1`
Chrome version: `115.0.5790.110 (Official Build) (64-bit)`
Operating system:
```
Edition	Windows 10 Pro
Version	21H2
Installed on	‎4/‎24/‎2021
OS build	19044.3086
Experience	Windows Feature Experience Pack 1000.19041.1000.0
```

## Description

Howdy, noticing the below bug when trying to run the windows autofill executable. For each prompt, I just hit enter to skip/continue. The program stalls here, having created the output folder `cards` but does not populate it or advance beyond this stage. Any advice?

PS: at the bottom of this post is the .xml the application detects and attempts to process.

Error log:
```
Successfully parsed card order: C:\Users\Evan\Downloads\animar-cards.xml
Your order has a total of 100 cards, in the MPC bracket of up to 108 cards.
(S30) Standard Smooth cardstock (nonfoil).

An uncaught exception occurred: No such driver version 115.0.5790.110 for win32
Press Enter to exit.
Images Uploaded   0%|                                                                          |  0/91 [00:00<?, 0.00/s]Images Downloaded   0%|                                                                        |  0/91 [00:00<?, 0.00/s]State: Initialising, Action: N/A 
```

`animar-cards.xml` contents:
```
<order><details><quantity>100</quantity><bracket>108</bracket><stock>(S30) Standard Smooth</stock><foil>false</foil></details><fronts><card><id>1HWTUq1uKbkrZspuP2eZzrWqISvcbJr1o</id><slots>0</slots><name>Wood Elves.png</name><query>wood elves</query></card><card><id>1JXi7L94TZk6hitNfF_Vbd7bMbcWwApZ_</id><slots>1</slots><name>Gwenna, Eyes of Gaea.png</name><query>gwenna eyes of gaea</query></card><card><id>1IfoCxVS-XnSFUxGHE8zcIIYLROXTGCy_</id><slots>2</slots><name>Ashnod's Altar.png</name><query>ashnods altar</query></card><card><id>1x7atj1Irnx-IfNtehRmq6o8NhSIqdsj0</id><slots>3</slots><name>Sol Ring (SLD).jpg</name><query>sol ring</query></card><card><id>1428MEsNiSg_0HHpP7AhA6kKCDWMtgMAd</id><slots>4</slots><name>Rattleclaw Mystic.png</name><query>rattleclaw mystic</query></card><card><id>1JvVe0iBR7Mf9VsCNMqsRGPLEgSTCb4zS</id><slots>5</slots><name>Ornithopter of Paradise (Showcase).jpg</name><query>ornithopter of paradise</query></card><card><id>1yAJekDQsCPx8kw9Ru1J-a84v58K1YUaK</id><slots>6</slots><name>Soul of the Harvest.png</name><query>soul of harvest</query></card><card><id>1PjSuT-V1qO1xTTRkr5piDMICO03y5DwL</id><slots>7</slots><name>Kami of Whispered Hopes.png</name><query>kami of whispered hopes</query></card><card><id>1LQmBXS3xc3HPz8gfhAVAvWoZlnl5tw-S</id><slots>8</slots><name>Rhythm of the Wild.png</name><query>rhythm of wild</query></card><card><id>11lBWzZ6SZD3ZQd29sAk9cE-TQC1Rmcjf</id><slots>9</slots><name>Pathbreaker Ibex (SLD).jpg</name><query>pathbreaker ibex</query></card><card><id>1NF8cHDBVx7CNKc8edI-4oO48kqC-oInu</id><slots>10</slots><name>Artisan of Kozilek.png</name><query>artisan of kozilek</query></card><card><id>19J2hdqCF_Cw07lyRGK8n-LisSNPuSh9i</id><slots>11</slots><name>Urabrask the Hidden.png</name><query>urabrask hidden</query></card><card><id>14eifGerihGs8Ilp7IlwkLHT2zLbEB8Qu</id><slots>12</slots><name>Ancestral Statue.png</name><query>ancestral statue</query></card><card><id>16Wso6lIsRAA1fRd6aYj3DvigPPsRSxco</id><slots>13</slots><name>Thrummingbird (Modern Efrem Palacios).jpg</name><query>thrummingbird</query></card><card><id>17cXtRUyeqSYsOL-qlipRAHVunGFbYbaa</id><slots>14</slots><name>Ulvenwald Tracker (Showcase Short Aya Kakeda).jpg</name><query>ulvenwald tracker</query></card><card><id>1GGnrRYv3oU0cGnvehOuzFBFQ6O6lUydg</id><slots>15</slots><name>Vorinclex, Voice of Hunger.png</name><query>vorinclex voice of hunger</query></card><card><id>1R7iGWY6WgkvlK_gGkRB-YDO4hHd1qL3N</id><slots>16</slots><name>Bone Sabres (NormalPlus David Auden Nash).jpg</name><query>bone sabres</query></card><card><id>1-bKLZMZYquvuf7K5pp2m681TQ4k-J645</id><slots>17</slots><name>Ancient Silver Dragon (Showcase Andrey Kuzinskiy).jpg</name><query>ancient silver dragon</query></card><card><id>1WVEhfBhaO128zU8c2ZGrfG9Bv3UKpTXI</id><slots>18</slots><name>Sakashima of a Thousand Faces (2XM Box Topper 6).png</name><query>sakashima of a thousand faces</query></card><card><id>1v2oJcO7r-qvt7cmquUXuNmid42RBDuOK</id><slots>19</slots><name>Etali, Primal Storm (Midjourney).jpg</name><query>etali primal storm</query></card><card><id>1q_Ab0GhsjIydri8IyGKR92naK1qTn5W6</id><slots>20</slots><name>Talisman of Creativity.png</name><query>talisman of creativity</query></card><card><id>1-3y5a3-jk36NyzOySgWlXykeVelCdiSq</id><slots>21</slots><name>Ancient Bronze Dragon (Showcase Raoul Vitale).jpg</name><query>ancient bronze dragon</query></card><card><id>11VJVTIYoTLL4IsLKTBNtECYsbVmJS3x2</id><slots>22</slots><name>Reclamation Sage (Christopher Moeller).jpg</name><query>reclamation sage</query></card><card><id>1R1dJ3KiCUAt-t_SES0d94y7jSQ052DC4</id><slots>23</slots><name>Benevolent Hydra (NormalPlus Eric Deschamps).jpg</name><query>benevolent hydra</query></card><card><id>1yAhswak3K9fva9Rql4Yrc4sALno3dKfA</id><slots>24</slots><name>Hydra's Growth (b).png</name><query>hydras growth</query></card><card><id>1e4MLjDB18-xpzPcRxlMXf4LMv2INHn8f</id><slots>25</slots><name>Selvala, Heart of the Wilds (Borderless Minimalist Tyler Jacobson).jpg</name><query>selvala heart of wilds</query></card><card><id>1bJGPyV1OpA_jZGJSDwVWqTNPIWkQ0oUQ</id><slots>26</slots><name>Fellwar Stone.jpg</name><query>fellwar stone</query></card><card><id>1tqW8IRSja-DkohwlqhL5MWjAs3jLm8gp</id><slots>27</slots><name>Birds of Paradise (Extended Eli,age 8).jpg</name><query>birds of paradise</query></card><card><id>1Y3-w9UQHRrgnublwjLuoN-Zp5WRqQA-1</id><slots>28</slots><name>Llanowar Elves.png</name><query>llanowar elves</query></card><card><id>1PafNLwdsFa0Kzjp-WWibA3yZw-KjuQMT</id><slots>29</slots><name>Talisman of Impulse.png</name><query>talisman of impulse</query></card><card><id>156wTAgcz2UkYF7MrmJLDI9dN9OHE4wE1</id><slots>30</slots><name>Chromatic Lantern.png</name><query>chromatic lantern</query></card><card><id>1iBHqGNlfQ7bBR8Q7Fw3-PuCo9cegpm0l</id><slots>31</slots><name>Hardened Scales (b).png</name><query>hardened scales</query></card><card><id>1vtLexk1-icQEWNDX4KEsl6KyT9VB6n8y</id><slots>32</slots><name>Vorel of the Hull Clade.png</name><query>vorel of hull clade</query></card><card><id>1X15RkqizFTp17fh6BEnUvIPLfqQazfnl</id><slots>33</slots><name>Wandering Archaic.jpg</name><query>wandering archaic</query></card><card><id>1Sl5tb-Zz5MmerCy-jW73E-BwjHaIH9EG</id><slots>34</slots><name>Jin-Gitaxias, Core Augur (Shifted).jpg</name><query>jin gitaxias core augur</query></card><card><id>1BTC1nmdJZzp1bQCnhLzXactPg7LbQj6P</id><slots>35</slots><name>Hydroid Krasis.png</name><query>hydroid krasis</query></card><card><id>1XvE-B8BwY-4MfBGGC4K5vezEh-jYdizr</id><slots>36</slots><name>Broodlord (Tyranids).png</name><query>broodlord</query></card><card><id>1lwjwKPZ65yE1lE1zE507yzumtY5cJyRx</id><slots>37</slots><name>Elvish Mystic.png</name><query>elvish mystic</query></card><card><id>1xfUQYUrlHYIYyVxrOnDhRoG-usiRfGc4</id><slots>38</slots><name>Arcane Signet (Extended Dan Scott).jpg</name><query>arcane signet</query></card><card><id>1_1b3ZfawVpsC8-teVAFj89waYnJo1HdE</id><slots>39</slots><name>Fyndhorn Elves (Classic Promo Igor Kieryluk).jpg</name><query>fyndhorn elves</query></card><card><id>1Ptii1vyKmoA7I_Ol9FUWAhpMO9R31WVx</id><slots>40</slots><name>Shivan Devastator (Showcase Brent Hollowell).jpg</name><query>shivan devastator</query></card><card><id>1-8WeUs2AG84HoiAzDSacTAonnKH9Xms1</id><slots>41</slots><name>Eternal Witness.png</name><query>eternal witness</query></card><card><id>1n8OvKpP_4opCqrsbcDOLQDsxn1G58JKU</id><slots>42</slots><name>Ozolith, the Shattered Spire.png</name><query>ozolith shattered spire</query></card><card><id>1VtDlucXgWzt98BIubbg0juh2TLusm17f</id><slots>43</slots><name>Inga and Esika (a).png</name><query>inga and esika</query></card><card><id>1BTfsRQ3AwHs3NdQCRT56M506vNuaWcd3</id><slots>44</slots><name>Primal Surge.png</name><query>primal surge</query></card><card><id>190-sePYo7jiZxAPmV-HKNNeoJYSMG1yJ</id><slots>45</slots><name>Purphoros, God of the Forge (Stargazing).png</name><query>purphoros god of forge</query></card><card><id>1VtjPgJ2d9ZDBds8S4v3LEEVeH7WLxFvV</id><slots>46</slots><name>Sylvan Library (Squirrel).png</name><query>sylvan library</query></card><card><id>1S_OSJPNGHvimXkWrtvViBXveU5kvJZOe</id><slots>47</slots><name>Spellskite.png</name><query>spellskite</query></card><card><id>1zDd314c9-UN2-v_frAaciM8CZ3SreqnS</id><slots>48</slots><name>Caustic Caterpillar.png</name><query>caustic caterpillar</query></card><card><id>184xpBwpU6ITnW-HkxvQg1FXsR7HNfvnk</id><slots>49</slots><name>Swiftfoot Boots (c).png</name><query>swiftfoot boots</query></card><card><id>19q2RK1SJ99tpeA9qp1IjQffB0HSJDACD</id><slots>50</slots><name>Surrak Dragonclaw.png</name><query>surrak dragonclaw</query></card><card><id>1uMukVPU2CV4PaoZK7iCsAbVW3WT4z8jq</id><slots>51</slots><name>Survival of the Fittest (b).png</name><query>survival of fittest</query></card><card><id>1orIDpKJWNNLLuz-U-9qW1pwZG1Aog_V9</id><slots>52</slots><name>Forgotten Ancient.png</name><query>forgotten ancient</query></card><card><id>19Eb66Vi6q5HO0ECir3SLKJrRnt2o68Sq</id><slots>53</slots><name>Talisman of Curiosity.png</name><query>talisman of curiosity</query></card><card><id>1IAt2UNQhDaERoC3UVs2CJsorff1s6_e-</id><slots>54</slots><name>Vizier of the Menagerie (Showcase Omar Rayyan).jpg</name><query>vizier of menagerie</query></card><card><id>1VCf4gbh1INToSc9dCmxbkqLMvkFEn-hI</id><slots>55</slots><name>Impact Tremors.png</name><query>impact tremors</query></card><card><id>1tnZ3YwbMGsUxEyOORTvt5Pf-OV6r6NBy</id><slots>56</slots><name>Commander_s Sphere (b).png</name><query>commanders sphere</query></card><card><id>1FCz85nY5TdTnEXzArkmQ-mvt-nJEiMH5</id><slots>57</slots><name>Deepglow Skate.png</name><query>deepglow skate</query></card><card><id>1zsj_CAp0g9HJ8s3DA2OYHD59qjSxY4xw</id><slots>58</slots><name>Beast Whisperer.png</name><query>beast whisperer</query></card><card><id>1OoThU-rPzcZ5EM1HyZNQjSzNekWNwidt</id><slots>59</slots><name>Zoanthrope (NormalPlus Billy Christian).jpg</name><query>zoanthrope</query></card><card><id>1wnB01qBMEfUzW6PwIAhBtEjgTr6WXtdm</id><slots>60</slots><name>The Ozolith.png</name><query>ozolith</query></card><card><id>1AYsBcNwcrueCFgJ9brzmb00AZjgVG26N</id><slots>61</slots><name>Myr Scrapling.jpg</name><query>myr scrapling</query></card><card><id>1OvZMu2B_uMBOuXue9sf300Nfzx6ZXFgy</id><slots>62</slots><name>Aberrant (cropped art).png</name><query>aberrant</query></card><card><id>1zGtWOBDSFJDXEhDaO9xWK50arNTlikIl</id><slots>63</slots><name>Lightning Greaves (2XM Showcase).png</name><query>lightning greaves</query></card><card><id>1hpHBIpFPTKF6Bi3p9Q7JFDvreO_Vnvd2</id><slots>64</slots><name>Tyrant Guard.png</name><query>tyrant guard</query></card><card><id>1prR6OlS2zyMuh0nlpnbSGs5ihaVZ2NpH</id><slots>65</slots><name>Consecrated Sphinx (MP2).png</name><query>consecrated sphinx</query></card><card><id>15qDCTUUn8cjdHUJL52juW5mLVTkl94Dz</id><slots>66</slots><name>Steam Vents (Extended).png</name><query>steam vents</query></card><card><id>19e9yTT7AgDttQCi2QMmqfyvCChce-XLO</id><slots>67</slots><name>Spire Garden (Extended).png</name><query>spire garden</query></card><card><id>19BaiO8r87OIKKongAhdEEA5NFKkZy9en</id><slots>68</slots><name>Training Center (Extended).png</name><query>training center</query></card><card><id>19Q3N4_nIKvdanLmxcNtNUJjyA650SMpm</id><slots>69</slots><name>Rejuvenating Springs (Extended).png</name><query>rejuvenating springs</query></card><card><id>166KCgF7erUnXTwnHla8md6cWOOu3ZI4A</id><slots>70</slots><name>Breeding Pool (Extended).png</name><query>breeding pool</query></card><card><id>16VxkpHZhXnjWZE09cfy3pxQtPdkmW_Vj</id><slots>71</slots><name>Stomping Ground (Extended).png</name><query>stomping ground</query></card><card><id>1-hwIs3kjkoH-7niWp-wNsoO_yefN_cM-</id><slots>72</slots><name>Frontier Bivouac.png</name><query>frontier bivouac</query></card><card><id>1y9sfhdlx44dupNha6OU40D_vThLObldL</id><slots>75,76</slots><name>Forest (25).jpg</name><query>forest</query></card><card><id>1KkXxg2uHgSl0g14Poqw87i8CUNag77uQ</id><slots>77,87</slots><name>Forest (6).jpg</name><query>forest</query></card><card><id>1SKjkP_-bJbMRjSgSFh2IAnN6beuE8DNi</id><slots>88,89</slots><name>Forest (2).jpg</name><query>forest</query></card><card><id>1w353hbTPJOaiFa9xyzypxE6zYK1o9c0S</id><slots>73,74</slots><name>Forest (22).jpg</name><query>forest</query></card><card><id>1c4YpFYinhiItDRaPNZWgz-Z-6UCKWf9Y</id><slots>90,91</slots><name>Forest (26).jpg</name><query>forest</query></card><card><id>1tgWAnq0wgV1qx6yvJy4Iwfy8eKj7fTGV</id><slots>82,83</slots><name>Island (34).jpg</name><query>island</query></card><card><id>1Q06eRtsbcaZT2pwcPhQ4XlkIhQRabhwu</id><slots>78,79</slots><name>Island (9).jpg</name><query>island</query></card><card><id>10cuI6GGIbqv0KYEItF8vbYVDj0Wt1giH</id><slots>80,81</slots><name>Island (44).jpg</name><query>island</query></card><card><id>1rYok5dsqyC9mpPeRYFaohpt6JPCTGH38</id><slots>86,92</slots><name>Mountain (Hasui Kawase 3).png</name><query>mountain</query></card><card><id>1VrBckfks7csN3OAVxOJtbE256ddMEFXK</id><slots>93,94</slots><name>Mountain (Hasui Kawase 2).png</name><query>mountain</query></card><card><id>1sw2P77u78A9ADCgS6WgYtgCBQQUgIXS0</id><slots>84,85</slots><name>Mountain (Hasui Kawase).png</name><query>mountain</query></card><card><id>1R7mP7aLQKNNiprWP6egIQYkyG8NGXbiE</id><slots>95</slots><name>Ancient Ziggurat.png</name><query>ancient ziggurat</query></card><card><id>1ILw4AlCVRT0COqlR0vqN5NY7ENTZj0wJ</id><slots>96</slots><name>Command Tower (Ryan Yee).jpg</name><query>command tower</query></card><card><id>190iiP_a9cBnWp1v_rDl2hYCBXAjydJ00</id><slots>97</slots><name>Cinder Glade (Extended).png</name><query>cinder glade</query></card><card><id>1NOpsHCD-jVcxiyE2-fLCe4IZFkLL7NCA</id><slots>98</slots><name>Ketria Triome.png</name><query>ketria triome</query></card><card><id>128SUJ81EvT-_BNcxi8X4v_3-dQ-o_Txb</id><slots>99</slots><name>Animar, Soul of Elements (b).png</name><query>animar soul of elements</query></card></fronts><backs><card><id>1tDc_GbIwRNYuDBUMhnV-TU-7vBWzBGd-</id><slots>33</slots><name>Explore the Vastlands.jpg</name><query>explore vastlands</query></card></backs><cardback>15YWveDA50fUYSDciXmef_47lyere9ntF</cardback></order>
```

**Comment by nolasaint** [comment]
Seems others are having this issue as well, per [r/mpcproxies](https://www.reddit.com/r/mpcproxies/comments/159vm6e/no_such_driver_version_11505790110_for_win32/).

**Comment by ndepaola** [comment]
g'day - the library we use for webdriver management (`webdriver_manager`) seems to still be having issues with chrome versioning on the latest release. related thread in that repo: https://github.com/SergeyPirogov/webdriver_manager/issues/566 at the moment i don't have a solution other than monitoring the situation and issuing a new release of the tool when it looks like it's properly been fixed in `webdriver_manager`. sorry for the inconvenience!!

in the short-term, you could try:
1. reverting your chromedriver to a previous version - try version 114, and if you get a similar error message to this on that chrome version, additionally revert to the previous release of our desktop tool, or
2. running the tool through brave or edge

**Comment by nolasaint** [comment]
Thanks for the quick reply! I'll see about using a different browser, likely the easiest path forward :)

**Comment by ndepaola** [comment]
seems like it may be fixed in 4.0.0 of `webdriver_manager` here https://github.com/SergeyPirogov/webdriver_manager/releases/tag/v4.0.0 going to test a bit and will push a new release if it all looks good

**Comment by ndepaola** [comment]
should be fixed in https://github.com/chilli-axe/mpc-autofill/releases/tag/v4.3.2 pls reopen this issue if it persists for you!
